content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -17,6 +17,8 @@ The following [Preview](/about/prisma/releases#preview) feature flags are availa
 - [`filteredRelationCount`](/concepts/components/prisma-client/aggregation-grouping-summarizing#filter-the-relation-count) since release [4.3.0](https://github.com/prisma/prisma/releases/tag/4.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15069)
 - [`fieldReference`](/reference/api-reference/prisma-client-reference#compare-columns-in-the-same-table) since release [4.3.0](https://github.com/prisma/prisma/releases/tag/4.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15068)
 - [`multiSchema`](https://github.com/prisma/prisma/issues/1122#issuecomment-1231773471) since release [4.3.0](https://github.com/prisma/prisma/releases/tag/4.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15077)
+- [`deno`](/guides/deployment/deployment-guides/deploying-to-deno-deploy) since release [4.53.0](https://github.com/prisma/prisma/releases/tag/4.5.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15844)
```

Comment:
```suggestion
- [`deno`](/guides/deployment/deployment-guides/deploying-to-deno-deploy) since release [4.5.0](https://github.com/prisma/prisma/releases/tag/4.5.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15844)
```

Reply:
Uhhh good catch!